src/state/swaps/swapsStore.ts

```diff
@@ -27,6 +27,9 @@ export interface SwapsState {
   selectedOutputChainId: ChainId;
   outputSearchQuery: string;
 
+  percentageToSell: number; // Value between 0 and 1, e.g., 0.5, 0.1, 0.25
```

Comment:
when a user passes in a `percentageToSell ` like `0.123131231321321` should we convert it to `0.12` and so on?

Reply:
should we? I don't see percentage to sell as something to worry about precision, but shouldn't convert to `0.12` rn

Reply:
just a question, up to you whether you deem it necessary. I personally think it's fine.